packages/@lightningjs/ui-components/src/docs/ThemingMode.mdx

```diff
@@ -54,7 +50,7 @@ componentConfig: {
     styleConfig: {
       mode: {
         focused: {
-          radius: 100
+          radius: 100;
```

Comment:
remove the semi-colon since these are in objects

Reply:
again, linter is auto correcting